src/gobprepare/importers/csv_importer.py

```diff
@@ -85,7 +85,7 @@ def _process_chunk(self, df: pd.DataFrame) -> tuple[list[str], list[tuple[Option
         # Replace empty values with None to insert NULL in database
         df.replace(to_replace=pd_NA, value=None, inplace=True)
 
-        columns = [self._column_names.get(col, col) for col in df if isinstance(col, str)]
+        columns = [self._column_names.get(col, col).strip() for col in df if isinstance(col, str)]
```

Comment:
Can you add a test for this case? (ie "column_name ")

Reply:
Done!